apps/src/code-studio/pd/foorm/editor/form/FormEditorManager.jsx

```diff
@@ -106,31 +104,33 @@ class FormEditorManager extends React.Component {
           hasLoadError: true
         });
       });
-  }
+  };
 
-  initializeEmptyCodeMirror = () => {
-    this.props.setLastSaved(null);
-    this.props.setSaveError(null);
-    this.updateFormData({
+  // For loading buttons
+  resetSelectedData() {
+    this.props.setFormData({
       questions: {},
       published: null,
       formName: null,
       formVersion: null,
       formId: null
     });
-    this.setState({
-      showCodeMirror: true,
-      hasLoadError: false
-    });
-  };
+  }
+
+  // For loading buttons
+  showCodeMirror() {
+    this.setState({showCodeMirror: true});
+  }
 
+  // For loading buttons
```

Comment:
doesn't look like this is being used as a callback anymore, so this comment is misleading